contrib/scripts/ast_coredumper

```diff
@@ -1,73 +1,78 @@
-#!/usr/bin/env bash
+#!/bin/bash
+
 # Turn on extended globbing
 shopt -s extglob
 shopt -s nullglob
 # Bail on any error
 set -e
 
-prog=$(basename $0)
+prog=$(basename "$0")
 
 # NOTE: <(cmd) is a bash construct that returns a temporary file name
 # from which the command output can be read.  In this case, we're
 # extracting the block of text delimited by '#@@@FUNCSSTART@@@'
 # and '#@@@FUNCSEND@@@' from this file and 'source'ing it to
 # get some functions.
-source <(sed -n -r -e "/^#@@@FUNCSSTART@@@/,\${p;/^#@@@FUNCSEND@@@/q}" $0 | sed '1d;$d')
+# shellcheck disable=SC1090
+source <(sed -n -r -e "/^#@@@FUNCSSTART@@@/,\${p;/^#@@@FUNCSEND@@@/q}" "$0" | sed '1d;$d')
```

Comment:
And I might be wrong but I feel like this could be simplified to:

    source <(sed -n "/^#@@@FUNCSSTART@@@/,/^#@@@FUNCSEND@@@/ p" "$0" | sed '1d;$d')

Reply:
Yep, that seems to work fine.